tests/test_validation_framwork/test_data_manager.py

```diff
@@ -46,6 +46,7 @@
 
 class TestDatasetRuntimeError(object):
     """Test dataset that acts as a fake object for the base classes."""
+    __test__ = False # prevent pytest from collecting this class during testing
```

Comment:
This results in classes named ``Test....`` not being collected by pytest, which would otherwise print a warning